tools/targets/NCS.py

```diff
@@ -48,17 +51,41 @@ def add_fib_at_start(arginput):
     regions = len(start_end_pairs)
 
     if regions == 1:
+        #single range indicating fits within first flash block (<320K)
         start, end = start_end_pairs[0]
+        print("Memory start 0x%08X, end 0x%08X" % (start, end))
```

Comment:
@0xc0170 Functionally, the print statements do not hurt anything, but for consistent user experience perhaps in the future they could be turned off by default.  There can be a macro in the post build script that could enable / disable them.  

Can this PR be merged as-is and then the issue of extra prints be addressed this later?